File: Food_Delivery_Core/Food_Delivery_Core/DTO/RegisterDto.cs
namespace Food_Delivery_Core.DTO
{
    public class RegisterDto
    {

        public string Username { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }

        public string Email { get; set; }
        public string FullName { get; set; }
        public string Adress { get; set; }
        public DateTime DoB { get; set; }
        public string UserType { get; set; }



    }
}

File: Food_Delivery_Angular/src/app/Models/RegisterDTO.ts


export class RegisterDTO {
    id : number;
    username : string;
    password : string;
    confirmPassword : string;
    
    email : string;
    fullName : string;
    adress : string;
    doB : Date;
    userType : string;
    isVerified : boolean;

    constructor(username:string, password :string, conf:string, mail:string, fname : string, adress:string, date:Date, utype:string, ver:boolean) {
        this.id = 0;
        this.username = username;
        this.password = password;
        this.confirmPassword = conf;
        this.email = mail;
        this.fullName = fname;
        this.adress = adress;
        this.doB = date;
        this.userType = utype;
        this.isVerified = ver;

    }

}